src/utils/module.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '@react-native/normalize-colors' {
```

Comment:
Frustratingly `@react-native/normalize-colors` doesn't have a [DefinitelyTyped](https://github.com/DefinitelyTyped/DefinitelyTyped) package. I'll try and chase that up, but this will work for now.

Reply:
Looks like CI isn’t happy about it. Will take another look in the morning.